### PR TITLE
Common units

### PR DESCRIPTION
**Pull request by tisidro**
#Description
This is a documentation of learning as I work through a [Zastrow CSS tutorial](https://www.digitalocean.com/community/tutorials/how-to-work-with-the-box-model-in-css)
##Validation
- [ ] Code is proofread.
- [ ] Documents are formatted.
- [ ] Commit messages are appropriate.
Thank you for helping with my progress in Github workflow and CSS mastery! :)

### CODE REVIEW

**Comment by tisidro** [comment]
Megan, should I have squashed this first?

**Comment by mlogsdon18** [comment]
Totally up to you about squashing. I get self-conscious and like to clean mine up a bit beforehand but it's pretty typical to squash after you create the PR. You could definitely clean up some of the repetitive commit messages and squash those together.

File: 1apply-css/about.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <title>About Sharks</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <div>There are over <strong>500</strong> species of sharks.</div>
+    <div>The great white shark can grow up to <strong class="highlight">6.1 meters</strong> long.</div>
+</body>
+
+</html>
```
Comment (mlogsdon18) [inline 1apply-css/about.html]: I noticed this "no newline at the end of the file" is being flagged. It's best practice to have a newline at the end of your files and you can actually set VSCode up to do it automatically! [Here's a link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)

File: 1apply-css/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<html>
+  <head>
+    <title>
+      Terri's Examples: Applying CSS to HTML With Cascade and Specificity
+    </title>
+    <link rel="stylesheet" href="1styles.css" />
+  </head>
+
+  <body>
+<div class="main-container">
```
Comment (mlogsdon18) [inline 1apply-css/index.html]: Indent this to show that it falls within the body

File: 1apply-css/index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    </p>
+      <div class="main__paragraph"> <strong class="highlight">Finny</strong>, our other cat, will wait
+      patiently without meowing and <strong>scratching</strong> at the door.</div>
+     
+      <figure class="main__image">
+        <img src="images/shark.jpg"
+     alt="Cartoon depicting Sammy the Shark"
+     title="Sammy the Shark" />
+    <figcaption>The original example in our lesson featured "Sammy the Shark!"</figcaption>
+</figure>
```
Comment (mlogsdon18) [inline 1apply-css/index.html]: Just another layout/indent issue here

File: 5common-units/index.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	<body>
+        		<figure class="excerpt">
+			<h3 class="title">Advice from a Caterpillar</h3>
+			<blockquote class="quote">
+				<p class="quote-text">The Caterpillar was the first to speak.</p>
+				<p class="quote-text">“What <span class="large">size</span> do you want to be?” it asked.</p>
+				<p class="quote-text">“Oh, I'm not particular as to <span class="large">size</span>,” Alice hastily replied; “only one doesn't like changing so often, <span class="small">you know</span>.”</p>
+			</blockquote>
+			<figcaption class="citation">Excerpt from <cite>Alice's Adventures in Wonderland</cite> by Lewis Carroll.</figcaption>
+		</figure>
```
Comment (mlogsdon18) [inline 5common-units/index.html:20]: Just a few extra tabs here :)

**Review by mlogsdon18 — COMMENTED** [review]
Great work! My only callouts are some layout/indentation issues. Super minor but it always helps to keep your code organized so others can easily read it.

File: 5common-units/index.html (hunk 4 above)
Comment (tisidro) [reply 5common-units/index.html:20]: I think I forgot to "format document." I need to get into that habit! :)

File: 1apply-css/about.html (hunk 1 above)
Comment (tisidro) [reply 1apply-css/about.html]: This is PURE GOLD--thank you so much!!! :)

**Review by mlogsdon18 — APPROVED** [review]
I went ahead and approved but it looks like there are still some spacing issues. The CSS file seems to have some places where you're using shorter indents than others so just try to keep it consistent as you keep going!

**Pull request merged**